src/Features/CSharp/Portable/CodeRefactorings/ConvertLocalFunctionToMethod/CSharpConvertLocalFunctionToMethodCodeRefactoringProvider.cs

```diff
@@ -114,6 +114,7 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
                 modifiers: new DeclarationModifiers(isStatic, isAsync: declaredSymbol.IsAsync),
                 returnType: declaredSymbol.ReturnType,
                 refKind: default,
+                isInitOnly: false,
```

Comment:
not really getting how methods are init-only.

Reply:
Currently, only accessors can be init-only. But accessors are methods.
We have a discussion queued to discuss init-only methods more generally, but that should not affect this PR (assume that we don't have generalized init-only methods).

I can make the parameter optional, so it's not in your face at most places where it could not legitimately be set to true. Would that resolve your concern?

---
In reply to: [422535858](https://github.com/dotnet/roslyn/pull/44077#discussion_r422535858) [](ancestors = 422535858)

Reply:
Isn't init a different type of accessor entirely. I.e. a property would have three potential accessors?  GetMethod/SetMethod/InitMethod?

Reply:
No, it's best thought of as a modifier on `set`. But we shortened `initonly set` to `init`.
https://github.com/dotnet/csharplang/blob/master/proposals/init.md

Reply:
got it 👍   can you make it a default value?  thanks!